dev/Interop/StoragePickers/Microsoft.Windows.Storage.Pickers.idl

```diff
@@ -57,18 +65,20 @@ namespace Microsoft.Windows.Storage.Pickers
         String CommitButtonText;
         Windows.Foundation.Collections.IMap<String, Windows.Foundation.Collections.IVector<String> > FileTypeChoices{ get; };
         String DefaultFileExtension;
-        Windows.Storage.StorageFile SuggestedSaveFile;
+        String SuggestedSaveFilePath;
         String SuggestedFileName;
```

Comment:
I apparently missed this in the spec PR: How do `SuggestedSaveFilePath` and `SuggestedFileName` interact? (The answer appears to be in PickerCommon.cpp.)